File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/GeocodeResponse.java
package com.skedgo.tripkit.ui.geocoding;

import com.google.gson.annotations.SerializedName;
import com.skedgo.tripkit.common.model.Location;

import java.util.List;

public class GeocodeResponse {
    @SerializedName("query")
    private String mQuery;

    @SerializedName("choices")
    private List<Location> mChoiceList;

    public List<Location> getChoiceList() {
        return mChoiceList;
    }

    public String getQuery() {
        return mQuery;
    }
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/GeocodeResponse.kt
package com.skedgo.tripkit.ui.geocoding

import com.google.gson.annotations.SerializedName
import com.skedgo.tripkit.common.model.location.Location

data class GeocodeResponse(
    @SerializedName("query")
    val query: String,
    @SerializedName("choices")
    val choiceList: List<Location>
)